openapi/v2/support-notifications.yaml

```diff
@@ -57,7 +53,7 @@ components:
         id:
           type: string
           format: uuid
-        slug:
+        name:
```

Comment:
Please add description here and any other properties that are missing descriptions.

Since slug was removed and not replaced by name, what is this name?

Reply:
Oops, it should be removed.